src/main.js

```diff
@@ -55,8 +55,10 @@ if (window.location.search.indexOf("testEnvironment") > -1) {
      * navigator.language in ExtensionLoader (when making require contexts for each
      * extension).
      */
+    var urlLocale = window.location.search && /locale=([^&]+)&?/.exec(window.location.search);
+
     require.config({
-        locale: window.localStorage.getItem("locale") || (typeof (brackets) !== "undefined" ? brackets.app.language : navigator.language)
+        locale: !!urlLocale ? urlLocale[1] : (typeof (brackets) !== "undefined" ? brackets.app.language : navigator.language)
```

Comment:
NOTE: the `!!` isn't really necessary here, since `urlLocale` will exist or it won't, and be truthy/falsey as a result.